shell-completion/bash/systemd-cryptenroll

```diff
@@ -40,8 +40,8 @@ __get_tpm2_devices() {
 
 __get_block_devices() {
     local i
-    for i in /dev/*; do
-        [ -b "$i" ] && printf '%s\n' "$i"
+    for i in $(systemd-cryptenroll --list-devices); do
+        [ -L "$i" ] || printf '%s\n' "$i"
```

Comment:
why only show symbolic links? I'd not bother with such a restriction...

Reply:
Yeah this looks right to me

Reply:
I can tell you my personal experience as a bash-completion user. I have multiple vms with different layouts and I always forget which partitions are encrypted, so `--list-devices` is great. But, for me it's enough to suggest:
```
# systemd-cryptenroll /dev/
/dev/vda3  /dev/vdb   
```

rather than:

```
# systemd-cryptenroll /dev/
/dev/disk/by-diskseq/1-part3
/dev/disk/by-diskseq/2
/dev/disk/by-partuuid/7f2a89dc-60bc-46dc-b23c-89c299773c80
/dev/disk/by-path/pci-0000:04:00.0-part3
/dev/disk/by-path/pci-0000:04:00.0-part/by-partnum/3
/dev/disk/by-path/pci-0000:04:00.0-part/by-partuuid/7f2a89dc-60bc-46dc-b23c-89c299773c80
/dev/disk/by-path/pci-0000:04:00.0-part/by-uuid/975a16bc-9859-48cf-98c3-fecbe3e6f394
/dev/disk/by-path/pci-0000:08:00.0
/dev/disk/by-path/virtio-pci-0000:04:00.0-part3
/dev/disk/by-path/virtio-pci-0000:08:00.0
/dev/disk/by-uuid/975a16bc-9859-48cf-98c3-fecbe3e6f394
/dev/disk/by-uuid/a5db705e-50a1-4922-8dc0-aa2f961c74b1
/dev/vda3
/dev/vdb
```

Reply:
At least, by-path symlinks look useful to see what the block device is.

Reply:
But this is not a debugging tool? It's to make typing commands quicker. If multiple duplicated results are returned, it is actually slower.

Reply:
this really should include the symlinks. I do my testing generally on usb sticks, and on those the "by-id" are just awesome since they carry the vendor name, which is just so useful for usb sticks, since it makes very clear which one is which.

the symlinks are *good*, they are *useful*, they exist for a reason, they are a lot more useful than the completely opaque "/dev/sdc3" stuff.

Hence, please, do not hide the symlinks!

Reply:
ok, changed :)